tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -1,21 +1,34 @@
+from urllib.parse import urlparse
+
 import pytest
 from .common import *  # NOQA
+from .test_rke_cluster_provisioning import rke_config, engine_install_url, node_template_linode, \
+    validate_rke_dm_host_2
 
 namespace = {"p_client": None, "ns": None, "cluster": None, "project": None}
 
 
-def test_bkp_restore():
+def test_bkp_restore_local(create_project_client):
+    validate_backup_create_restore_delete("local")
+
+
```

Comment:
Define a variable to skip S3 related cases , since we dont want them to be added to onTag job .